tools/expotools/src/client-build/types.ts

```diff
@@ -0,0 +1,15 @@
+import aws from 'aws-sdk';
+import { Platform } from '../ProjectVersions';
+
+export { Platform };
+
+export type S3Client = aws.S3;
+
+export interface ClientBuilder {
+  platform: Platform;
+  getAppPath(): string;
+  getClientUrl(appVersion: string): string;
+  getAppVersionAsync(): Promise<string>;
+  buildAsync(): Promise<void>;
+  uploadBuildAsync(s3Client: S3Client, appVersion: string): Promise<void>;
+}
```

Comment:
```suggestion
export interface ClientBuilder {
  platform: Platform;
  getAppPath: () => string;
  getClientUrl: (appVersion: string) => string;
  getAppVersionAsync: () => Promise<string>;
  buildAsync: () => Promise<void>;
  uploadBuildAsync: (s3Client: S3Client, appVersion: string) => Promise<void>;
}
```
That's my preference when defining types for functions 😉 You don't have to change it here.